### ISSUE TITLE
[BUG] Analysis on tackle-testapp fails with error

### ISSUE THREAD

**Issue by sshveta** [konveyor/analyzer-lsp#1062]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Konveyor version

v0.9.0

### Priority

Major

### Current Behavior

Application : https://github.com/konveyor/tackle-testapp 
Mode: Source Analysis
Target": ["Jakarta EE 9", "Containerization"]

Analysis fails . 

Logs 
=============
```
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon (adapter) created.
time=2026-01-13T04:36:35Z level=info msg=[binding] |200|  GET /tasks/28
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  DELETE /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[binding] |201|  POST /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported started.
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=OptDir:    /addon/opt
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=SharedDir: /shared
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=CacheDir:  /cache
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=SourceDir: /shared/source
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=RuleDir:   /addon/rules
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=BinDir:    /shared/bin
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=M2Dir:     /cache/m2
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=Fetching application.
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[binding] |200|  GET /applications/6
time=2026-01-13T04:36:35Z level=info msg=[binding] |200|  GET /applications/6/identities
time=2026-01-13T04:36:35Z level=info msg=[binding] |200|  GET /settings/git.insecure.enabled
time=2026-01-13T04:36:35Z level=info msg=[binding] |200|  GET /proxies
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=[GIT] Created: /addon/.git/9b5ef7eb937e0cf4/.gitconfig
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=[GIT] Using identity: (id=7) wwUhmD
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=[GIT] Created: /addon/.git/9b5ef7eb937e0cf4/.git-credentials
time=2026-01-13T04:36:35Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:35Z level=info msg=[addon] Addon reported: activity. entry=[GIT] Using identity: (id=7) wwUhmD
time=2026-01-13T04:36:36Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:36Z level=info msg=[binding] |201|  POST /files/git.output
time=2026-01-13T04:36:36Z level=info msg=[addon] Addon reported: activity. entry=[CMD] Running: /usr/bin/git clone https://github.com/konveyor/tackle-testapp /shared/source/tackle-testapp
time=2026-01-13T04:36:36Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:36Z level=info msg=[addon] Addon attached. ID=443 activity=13 name=git.output
time=2026-01-13T04:36:36Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:36Z level=info msg=[addon] Addon reported: activity. entry=[CMD] /usr/bin/git failed: exit status 128
time=2026-01-13T04:36:36Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:36Z level=info msg=[binding] |204|  PATCH /files/443
time=2026-01-13T04:36:36Z level=error msg=[addon] Addon failed.
exit status 128
github.com/konveyor/tackle2-hub/command.(*Command).RunWith()
	/opt/app-root/pkg/mod/github.com/konveyor/tackle2-hub@v0.9.0-alpha.1.0.20251202133231-c937c30f820f/command/cmd.go:77
github.com/konveyor/tackle2-hub/command.(*Command).Run()
	/opt/app-root/pkg/mod/github.com/konveyor/tackle2-hub@v0.9.0-alpha.1.0.20251202133231-c937c30f820f/command/cmd.go:43
github.com/konveyor/tackle2-hub/scm.(*Git).Fetch()
	/opt/app-root/pkg/mod/github.com/konveyor/tackle2-hub@v0.9.0-alpha.1.0.20251202133231-c937c30f820f/scm/git.go:60
main.(*Mode).fetchRepository()
	/opt/app-root/src/cmd/mode.go:93
main.(*Mode).Build()
	/opt/app-root/src/cmd/mode.go:31
main.main.func1()
	/opt/app-root/src/cmd/main.go:91
github.com/konveyor/tackle2-hub/addon.(*Adapter).Run()
	/opt/app-root/pkg/mod/github.com/konveyor/tackle2-hub@v0.9.0-alpha.1.0.20251202133231-c937c30f820f/addon/adapter.go:146
main.main()
	/opt/app-root/src/cmd/main.go:57
runtime.main()
	/usr/lib/golang/src/runtime/proc.go:285
runtime.goexit()
	/usr/lib/golang/src/runtime/asm_amd64.s:1693
time=2026-01-13T04:36:36Z level=info msg=[addon] Addon reported: error. error={Severity:Error Description:exit status 128}
time=2026-01-13T04:36:36Z level=info msg=[binding] |204|  PUT /tasks/28/report
time=2026-01-13T04:36:36Z level=info msg=[addon] Addon reported: failed. reason=exit status 128
```

### How Reproducible

Always (Default)

### Environment

```markdown
- **Analyzer Image SHA** 
- **JDTLS  base image SHA**
- **Provider Settings JSON**
```
